models/WebsiteSetting.php

```diff
@@ -28,9 +28,9 @@ final class WebsiteSetting extends AbstractModel
 {
     protected ?int $id = null;
 
-    protected string $name;
+    protected string $name = '';
 
-    protected string $language;
+    protected string $language = '';
 
     protected string $type;
```

Comment:
```suggestion
    protected string $type = '';
```

Reply:
@blankse We make it nullable and set the default value to null, because in the database there is no default value and the `type` can be null